samples/core/parameterized_tfx_oss/parameterized_tfx_oss.py

```diff
@@ -13,168 +13,170 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import json
-import os
-
-import kfp
-import kfp.onprem
```

Comment:
We should make this sample work, as it's one of the pre-load sample pipelines shipped with KFP release. Removing this sample is actually the cause for the backend e2e test failure.
What was the issue if we use `kfp.deprecated` here?